Makefile

```diff
@@ -36,9 +51,25 @@ test: format gosec golint
 install:
 	install ./leaktk-scanner $(DESTDIR)$(PREFIX)/bin/leaktk-scanner
 
+.PHONY: install.completions
+install.completions:
+	install ${SELINUXOPT} -d -m 755 $(DESTDIR)${BASHINSTALLDIR}
+	install ${SELINUXOPT} -m 644 completions/bash/leaktk-scanner $(DESTDIR)${BASHINSTALLDIR}
+	install ${SELINUXOPT} -d -m 755 $(DESTDIR)${ZSHINSTALLDIR}
+	install ${SELINUXOPT} -m 644 completions/zsh/_leaktk-scanner $(DESTDIR)${ZSHINSTALLDIR}
+	install ${SELINUXOPT} -d -m 755 $(DESTDIR)${FISHINSTALLDIR}
+	install ${SELINUXOPT} -m 644 completions/fish/leaktk-scanner.fish $(DESTDIR)${FISHINSTALLDIR}
+
 security-report:
 	trivy fs .
 
 update:
 	go get -u ./...
 	go mod tidy
+
+.PHONY: validate.completions
+validate.completions: SHELL:=/usr/bin/env bash 
```

Comment:
Interesting. I didn't know you could specify the target twice like this. What does it do?

Reply:
Nor did I. It is in the Podman [Makefile](https://github.com/containers/podman/blob/main/Makefile#L616)